gpApi/apiFetch.js

```diff
@@ -0,0 +1,15 @@
+import { API_ROOT } from 'appEnv';
+
+export const apiFetch = async (url, revalidate = 3600) => {
```

Comment:
Might be clearer to name this something more descriptive for the intended purpose, like, `staticContentFetch` or something like that? 🤔